src/createActionArgs.js

```diff
@@ -40,17 +40,37 @@ var createStateArg = function (action, model, isAsync, compute) {
   return state
 }
 
+var createServicesArg = function (action, services, parentKey) {
+  return Object.keys(services).reduce(function (newServices, key) {
+    if (typeof services[key] === 'function') {
+      newServices[key] = function () {
+        action.services.push({
+          name: parentKey || 'root',
+          method: key,
+          args: [].slice.call(arguments)
+        })
+        return services[key].apply(services[key], arguments)
+      }
+    } else {
+      newServices[key] = createServicesArg(action, services[key], key)
```

Comment:
Hm, looking at the hoodie module you could still do that. Instead of:

```js
    services: {
      store: store,
      detach: function detach () {
        store.off('change', callbacks.onStoreChange)
        store.off('clear', callbacks.onStoreClear)
      }
    }
```

You would:

```js
    store

    services: {
      store: {
        func1: store.func1.bind(store),
        func2: store.func2.bind(store),
        getSomeValue: function () { return store.someValue; }
      },
      detach: function detach () {
        store.off('change', callbacks.onStoreChange)
        store.off('clear', callbacks.onStoreClear)
      }
    }
```

In my opinion it opens up new possibilities to do it like this. Cerebral will now understand how you use your services, which might not only be beneficial for the debugger, but could lead to other solutions as well. If we let services be "whatever", there is no way Cerebral can understand them :-/ I think we will miss out on possibly great things